data-reconciliation-app/demo/scripts/01-data_ingest_demo.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+sudo rm -rf ./demo/Reports/*.json
+VARS_PATH="./demo/scripts/env_vars.sh"
+. "$VARS_PATH"
+
+cd "${certs}"
+
+printf "Member1 trying to ingest data \n" 
+check_eq "Member1 - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg member0) -H "Content-Type: application/json" --data-binary "@../../demo/data/member1_demo_pt1.json" $only_status_code)"
+addCheckpoint "✅ Member1 successfully ingested data"
+
+# printf "Member2 trying to ingest empty data \n"
+check_eq "Member2 - data ingest failed (data is null or empty)" "400" "$(curl $ingestUrl -X POST $(cert_arg member1) -H "Content-Type: application/json" --data-binary "" $only_status_code)"
+addCheckpoint "❌ Member2 failed to ingest empty data"
+
+# printf "Member2 trying to ingest data \n" 
+check_eq "Member2 - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg member1) -H "Content-Type: application/json" --data-binary "@../../demo/data/member2_demo.json" $only_status_code)"
+printf "\n✅ Member2 successfully ingested data\n"
+
+# printf "Member3 trying to ingest data \n" 
+check_eq "Member3 - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg member2) -H "Content-Type: application/json" --data-binary "@../../demo/data/member3_demo.json" $only_status_code)"
+addCheckpoint "✅ Member3 successfully ingested data"
+
+
+curl $reportUrl -X GET $(cert_arg member0) |jq >> ../../demo/Reports/Member1_Ingest_Data_V1.json
```

Comment:
When I run this script - I get 
```
./demo/scripts/01-data_ingest_demo.sh: line 27: ../../demo/Reports/Member1_Ingest_Data_V1.json: No such file or directory
```

Reply:
Debugging it - if I create a "Reports" folder, the script works. Please add a `mkdir Reports -p` into the script so that it creates that folder if that is your intent

Reply:
Somehow it got missed, have added it now.